src/main/java/org/example/C01_10_MultiplicationOfIntegers.java

```diff
@@ -1,6 +1,6 @@
 package org.example;
 public class C01_10_MultiplicationOfIntegers {
-    public static int multiply(int num1, int num2) {
+    public static int multiplication(int num1, int num2) {
```

Comment:
Methods usualy represent actions so `multiply` is correct, `multiplication` is not.